wallet/res/layout/activity_main.xml

```diff
@@ -51,11 +51,4 @@
                 app:menu="@menu/bottom_navigation_menu" />
         </FrameLayout>
     </androidx.constraintlayout.widget.ConstraintLayout>
-
-    <de.schildbach.wallet.ui.widget.PleaseWaitRestoringWalletView
-        android:id="@+id/restoring_wallet_cover"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@color/background_primary"
-        android:visibility="gone" />
```

Comment:
This was part of the old invite system, it may not have worked.  It was to display something on the home screen telling the user to wait until syncing finished.